bin/lib/cli/builds.py

```diff
@@ -0,0 +1,165 @@
+import datetime
+import json
+import os
+import subprocess
+import sys
+import tempfile
+from collections import defaultdict
+from typing import Optional, Dict, Sequence
+
+import click
+import requests
+
+from lib.amazon import download_release_file, download_release_fileobj, find_latest_release, find_release, \
+    log_new_build, set_current_key, get_ssm_param, get_all_current, get_releases, remove_release, get_current_key, \
+    list_all_build_logs, list_period_build_logs
+from lib.cdn import DeploymentJob
+from lib.ce_utils import describe_current_release, are_you_sure, display_releases
+from lib.cli import cli
+from lib.env import Config
+from lib.releases import Version
+
+
+@cli.group()
+def builds():
+    """Build manipulation commands."""
+
+
+@builds.command(name="current")
+@click.pass_obj
+def builds_current(cfg: Config):
+    """Print the current release."""
+    print(describe_current_release(cfg))
+
+
+def old_deploy_staticfiles(branch, versionfile):
```

Comment:
this could probably be done separately, but we can definitely kill this now - all releases have used the new method for a long time and we aren't going back.

Reply:
Roger, will file an issue to remember